shell/platform/embedder/fixtures/main.dart

```diff
@@ -1364,3 +1364,21 @@ void render_gradient_retained() {
   };
   PlatformDispatcher.instance.scheduleFrame();
 }
+
+@pragma('vm:entry-point')
```

Comment:
What does this test do differently than the entrypoint you're replacing?

Reply:
The test executes a series of canvas draw calls that makes the `EntityPass` render a `Contents` that needs to obtain a `Pipeline` before the `ReactorGLES` is flushed.

Without this patch, the test hangs when `SolidColorContents::Render` tries to get its shader pipeline.